go/vt/vtgate/vstream_manager_test.go

```diff
@@ -987,17 +1008,34 @@ func TestResolveVStreamParams(t *testing.T) {
 		assert.Equal(t, wantFilter, filter, tcase.input)
 		require.False(t, flags.MinimizeSkew)
 	}
+
 	// Special-case: empty keyspace because output is too big.
-	input := &binlogdatapb.VGtid{
-		ShardGtids: []*binlogdatapb.ShardGtid{{
-			Gtid: "current",
-		}},
+	specialCases := []struct {
+		input  string
+		output string
+	}{
+		{
+			input:  "current",
+			output: "current",
+		},
```

Comment:
Do we still want/need the special `current` keyword and associated behavior? That seemed to be tied to "no keyspace value", which you altered here. Perhaps it makes sense to keep `current` thought if we use `/.*` instead of no value for the keyspace wildcard.

Reply:
I thought it was worth remaining the previous typical request to explicitly verify one of the main use cases even though my change tried to leave the case intact. But removing the irrelevant code in the latest code also makes sense. If I double down, I should add a comment about the background.
Anyway, as you mentioned, I'm revisiting this topic after settling the keyspace wildcard.

Reply:
Yeah, I think you're right. Let's get @rohit-nayak-ps's opinion too though.